### ISSUE TITLE
SampleActivity has leaked

### ISSUE THREAD

**Issue by RoRoche** [geronimoagency/CoffeeScene-Android#4]
Here is a memory leak found on CoffeeScene-Android:

```
In com.geronimostudios.sample.coffeescene:1.0:1.
* com.geronimostudios.sample.coffeescene.SampleActivity has leaked:
* GC ROOT static com.geronimostudios.coffeescene.SceneManager.sScenesMeta
* references java.util.Hashtable.table
* references array java.util.Hashtable$HashtableEntry[].[3]
* references java.util.Hashtable$HashtableEntry.key
* leaks com.geronimostudios.sample.coffeescene.SampleActivity instance
```

due to the static `sScenesMeta` map keeping the reference on the `Activity` and never release it.

**Comment by mradzinski** [comment]
Seems this was somehow solved by 94b6e4729b7401c6703af5522cf690f3423bef90 but never closed nor answered. Calling the `release` method on `onStop` or `onDestroy` should stop the Activity from leaking.

**Comment by jeromec-geronimo** [comment]
Thanks @mradzinski for your answer.
As said `SceneManager.release(this);` has to be called.
